src/Model.jl

```diff
@@ -216,4 +226,37 @@ end
 spin_components(model::Model) = spin_components(model.spin_polarization)
 
 
-_is_well_conditioned(A; tol=1e5) = (cond(A) <= tol)
+# prevent broadcast
+import Base.Broadcast.broadcastable
+Base.Broadcast.broadcastable(model::Model) = Ref(model)
+
+
+#=
+There are two types of quantities, depending on how they transform under change of coordinates.
+
+Positions transform with the lattice: r_cart = lattice * r_red. We term them vectors.
+
+Linear forms on vectors (anything that appears in an expression f⋅r)
+transform with the inverse lattice transpose: f_cart = lattice' \ f_red. We term them covectors.
+Examples of covectors are forces.
```

Comment:
I think it's a good idea to add the one line of arguments I've put into the forces function here.

Reply:
good idea